Tasks/Gradle/CodeAnalysis/SonarQube/common.ts

```diff
@@ -6,13 +6,15 @@ import fs = require('fs');
 
 import {ToolRunner} from 'vsts-task-lib/toolrunner';
 
+import tl = require('vsts-task-lib/task');
```

Comment:
In this proposal, where are the functions _isPrBuild()_, _getStagingDir()_ and _getRunSettings()_?